src/rendering/screenspacerenderable.cpp

```diff
@@ -434,11 +434,11 @@ ScreenSpaceRenderable::ScreenSpaceRenderable(const ghoul::Dictionary& dictionary
         // No sync or send because this is already inside a Lua script that was triggered
         // when this triggerProperty was pressed in the gui, therefor it has already been
         // synced and sent to the connected nodes and peers
-        global::scriptEngine->queueScript(
-            std::move(script),
-            scripting::ScriptEngine::ShouldBeSynchronized::No,
-            scripting::ScriptEngine::ShouldSendToRemote::No
-        );
+        global::scriptEngine->queueScript({
+            .code = std::move(script),
+            .synchronized = scripting::ScriptEngine::Script::ShouldBeSynchronized::No,
```

Comment:
Isn't this default?

Reply:
It seems like as part of this PR, it was set to sync per default.

Reply:
The default is that the scripts are synchronized and sendtoremote.  In this case we don't want it to happen since the `onChange` will already get triggered correctly through the scripting system